workerapi/client/cli.go

```diff
@@ -5,26 +5,30 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type CliClient interface {
+	Cli() error
+}
+
 type cliClient struct {
-	client
+	client  client
```

Comment:
This is actually cleaner, I realized, because the caller of NewCliClient doesn't care about the rpc calls so there's no need to expose those.